kuma/users/views.py

```diff
@@ -64,11 +64,17 @@ def ban_user(request, user_id):
                           is_active=True)
             ban.save()
             return redirect(user)
+    if user.active_ban:
+        return redirect(user)
     form = UserBanForm()
+    # A list of common reasons for banning a user
+    common_reasons = ['Spam', 'Profile Spam ', 'Sandboxing',
```

Comment:
I think the mechanism (a ``click`` event on an ``<a>`` tag) works, and adding this through the form would be a complication. It works like I'd expect in VM testing.

It is a good point that this value should be customizable. This is the part of this that I most expect to be adjusted once more people see it in production (no one comments on bugs, everyone has an opinion once they see it). Let's save the future PR and move the reasons to constance. [WIKI_DOCUMENT_TAG_SUGGESTIONS](https://github.com/mozilla/kuma/blob/master/kuma/settings/common.py#L1197-L1210) is a good model (``USER_BAN_REASONS`` for the name?).  You'll need to decode the JSON to turn it into a Python list again, and default to ['Spam'] if it is empty or has a decoding error (to avoid the alternate logic of no suggestions).